.github/workflows/scripts/test_disable_statements_check.py

```
@@ -0,0 +1,198 @@
+        self.assertFalse(self.checker._is_typescript_file('file.js'))
+
+    def test_auto_discovery(self):
+        """Test automatic method discovery."""
+        methods = []
+        for name, method in self.checker.__class__.__dict__.items():
+            if name.startswith('check_') and callable(method):
+                methods.append(name)
+        
+        expected_methods = ['check_eslint_disable', 'check_istanbul_ignore', 'check_it_skip']
+        for expected in expected_methods:
+            self.assertIn(expected, methods)
```

Ruff (0.14.8)

Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)